vc/issuer-manifest-credential-v1.jsonld

```diff
@@ -0,0 +1,23 @@
+{
+  "@context": {
+    "@version": 1.1,
+    "@protected": true,
+
+    "name": "http://schema.org/name",
+    "ex": "https://example.org/examples#",
+
+    "IssuerManifestCredential": {
+      "@id": "ex:IssuerManifestCredential",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "schema": "http://schema.org/",
```

Comment:
Aren't we missing extra fields here?

Reply:
this is name space to be used in 'contexts'